modules/test/playwright/playwright.config.ts

```diff
@@ -47,6 +47,7 @@ import {config as portalSearchWebConfig} from './tests/portal-search-web/config'
 import {config as portalSecurityScriptManagementWebConfig} from './tests/portal-security-script-management-web/config';
 import {config as portalSecurityServiceAccessPolicyService} from './tests/portal-security-service-access-policy-service/config';
 import {config as portalWorkflowKaleoDesignerWebConfig} from './tests/portal-workflow-kaleo-designer-web/config';
+import {config as portletConfigurationCssWebConfig} from './tests/portlet-configuration-css-web/config';
```

Comment:
This should be moved one line after for alphabetical sortering, eslint complains otherwise.